source/common/quic/quic_filter_manager_connection_impl.h

```diff
@@ -64,6 +64,9 @@ class QuicFilterManagerConnectionImpl : public Network::ConnectionImplBase,
     return network_connection_->connectionSocket()->connectionInfoProvider();
```

Comment:
Does this one need the same fix as well?

Reply:
I don't think this one is fixable, at least not without a larger refactor of this class (which maybe we should do?). This method returns a reference to a ConnectionInfoSetter. But if the network_connection_ or the network_connection_->connectionSocket() are null then this method will return a null reference (which is illegal). So presumably this method must only be called when they are non-null? If that is not a valid assumption then perhaps we need to refactor this class to make this information always available?

Reply:
Good point! Can you add ENVOY_BUG before returning?

Reply:
Done!